---
expected_error_type: none
---
// Class declaration inside function
fun foo() {
  class Superhero {}
  print "Class declared inside function";
}

foo();
print "Function called successfully";
